### ISSUE TITLE
Page not found at localhost:8089

### ISSUE THREAD

**Issue by mastercyb** [FactomProject/fctwallet#2]
I installed and synced factomd. factom-cli also works.
fctwallet is not working. 

![selection_009](https://cloud.githubusercontent.com/assets/410789/16074757/f3ee830a-32e3-11e6-93e0-e596673f7740.png)
![selection_010](https://cloud.githubusercontent.com/assets/410789/16074808/278c341e-32e4-11e6-9eb4-2022d18aa253.png)

I tried using my custom factomd.conf but it didnt help.
I didn't find any references on a similar error 
OS Ubuntu 16.04

**Comment by Emyrk** [comment]
FactomProject/fctwallet is not accessed through the browser. From your screenshots fctwallet is running as intended. If you wish to interact with it, one option is to use 'factom-cli' commands.

Run: 'factom-cli properties', if it succeeds then your fctwallet, factomd, and fct-wallet are running.

**Comment by carryforward** [comment]
Thank you for trying the system.

Sadly, with the current version of the fctwallet, there is no user friendly feedback that it is working correctly.  fctwallet is indented to serve an API that gets accessed by other programs.

Here is some documentation on how to access the current version of the API:
https://github.com/FactomProject/FactomDocs/blob/master/FactoidAPI.md

Both factomd and fctwallet expose APIs.  

There is a slightly working web display on factomd.  you can get to it on http://localhost:8090/controlpanel

https://github.com/FactomProject/FactomDocs/blob/master/developerSandboxSetup.md

We do have walletapp, which is a standalone program.  It can't run at the same time as fctwallet, since they share a wallet file.

Walletapp has a GUI user webpage that shows details of what is happening on the wallet.

**Comment by mastercyb** [comment]
Thanks guys for explanation. Do I understand correctly that in order for me to use GUI I need to run walletapp without fctwallet?

**Comment by carryforward** [comment]
yes.  they both use the same wallet database.  one locks it so the other one blocks.
